Source/Evolve/WarpXEvolve.cpp

```diff
@@ -159,7 +159,7 @@ WarpX::Evolve (int numsteps)
 
         // Main PIC operation:
         // gather fields, push particles, deposit sources, update fields
-
+        if (warpx_py_particleinjection) warpx_py_particleinjection();
```

Comment:
This looks good, though, for readability, can you add a blank line before and after this statement?

Reply:
Blank line added